modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -45,23 +45,28 @@ spec:
      outputRefs:
      - elasticsearch-secure <9>
      - default <10>
-   - name: infrastructure-audit-logs <11>
+     labels:
+       logs: application <11>
+   - name: infrastructure-audit-logs <12>
      inputRefs:
      - infrastructure
      outputRefs:
      - elasticsearch-insecure
+     labels:
+       logs: audit-infra
```

Comment:
I might add the annotation to this line, too, or add expand the description to acknowledge that each pipeline definition contains an optional label. (Please apply any changes that you make to this YAML to the other YAML examples that you've changed, except the last one.)